openadapt/strategies/mixins/svg_mixin.py

```diff
@@ -0,0 +1,43 @@
+"""
+Implements a ReplayStrategy mixin using Pypotrace to convert an image to SVG text.
+
+NOTE: additional libraries on the system are needed,
+        see https://github.com/flupke/pypotrace#installation
+
+Usage:
+
+    class MyReplayStrategy(SVGReplayStrategyMixin):
+        ...
+"""
+import subprocess
+import numpy as np
+import drawsvg as draw
+
+from openadapt.models import Recording, Screenshot
+from openadapt.strategies.base import BaseReplayStrategy
+
+
+class SVGReplayStrategyMixin(BaseReplayStrategy):
+    """ReplayStrategy mixin that uses Pypotrace to convert an image to SVG text.
+
+    Attributes:
+        recording: the recording to be played back
+
+    """
+    def __init__(self, recording: Recording):
+        super().__init__(recording)
+
+    def get_svg_text(self, screenshot: Screenshot) -> str:
+        input_img = screenshot.image
+        input_img.save("input.png")
+
+        subprocess.run(["vtracer", "--input", "input.png", "--output", "output.svg"])
```

Comment:
Can we please specify input.png and output.svg as configuration parameters? e.g. `config.SVG_INPUT_FILE_NAME` (or similar)

Also what do you think about using `tempfile` to store these in a dedicated temporary directory, in case clean-up fails?

Reply:
I created [NamedTemporaryFiles](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile) for both input and output files in [7d563f9](https://github.com/MLDSAI/OpenAdapt/pull/198/commits/7d563f99148ff1ab65d6b6b58c78c386fb4fab12), so as a result their file names are unable to be specified.